seqr/views/apis/saved_variant_api.py

```diff
@@ -270,6 +271,7 @@ def _variant_details(variant_json, user):
             'clinsig': extras.get('clinvar_clinsig'),
             'variantId': extras.get('clinvar_variant_id'),
             'alleleId': extras.get('clinvar_allele_id'),
+            'goldStars': extras.get('clinvar_gold_stars'),
```

Comment:
was thinking that could be added in a separate PR

Reply:
I'd rather we didn't return it to the client side yet then. It will still be saved to the saved variant json so we can access it later when we want to show it

Reply:
ok, I'll just add the ui

Reply:
done